jax-stuff.md

```diff
@@ -77,51 +78,52 @@ _styles: >
 
 ## How Does Parallelism Work in JAX?
 
-JAX supports two schools of thought for multi-device programming:
+JAX supports three levels of parallelism for multi-device programming:
 
-1. **Compiler, take the wheel!** Let the compiler automatically partition arrays and decide what communication to add to facilitate a given program. This lets you write a program on a single device and automatically run it on hundreds without changing anything. 
-2. **Just let me write what I mean, damnit!** While compilers are nice, they sometimes do the wrong thing and add communication you don't intend. Sometimes we want to be extremely explicit about what we're doing.
+1. **Compiler-based automatic parallelization** You program as if using a single global machine, and the compiler chooses how to shard data and partition computation (with some user-provided constraints).
```

Comment:
I kind of like the "Compiler, take the wheel" and "Just let me write what I mean, dammit" titles. They're not super descriptive, but I kind of like that they're colorful. Would you consider keeping them and making the third be "Compiler, take the wheel, but ask me when you don't know!"?

Reply:
Done.